sapi/fpm/fpm/fpm_status.c

```diff
@@ -385,6 +385,61 @@ int fpm_status_handle_request(void) /* {{{ */
 				full_post = "]}";
 			}
 
+			/* Prometheus */
+		} else if (fpm_php_get_string_from_table(_GET_str, "prometheus")) {
+			sapi_add_header_ex(ZEND_STRL("Content-Type: text/plain"), 1, 1);
```

Comment:
AFAIK prometheus is aiming to follow open metrics, therefore the content type should be different: https://github.com/OpenObservability/OpenMetrics/blob/master/specification/OpenMetrics.md#overall-structure

Reply:
```suggestion
			sapi_add_header_ex(ZEND_STRL("Content-Type: application/openmetrics-text; version=1.0.0; charset=utf-8"), 1, 1);
```